client/src/components/InfoPanel.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+  BoldBox, Container, GrayWrapper, InfoContainer, ModuleWrapper, LinkWrapper
+} from './styles/styles';
+import moment from 'moment';
+
+const InfoPanel = ({ infoPanel }) => (
+  <Container>
+    <ModuleWrapper>
+      <InfoContainer>
+        <GrayWrapper>
```

Comment:
Was going to suggest that it might be cleaner to map over the infoPanel array here but it seems you have different styled components for the first and last indices, you could map and add a couple if statements to save some lines of code? Overall, not necessary and may end up being just as much as this - looks solid overall